### PR TITLE
Guides: Containers: Podman: Migrate Quadlet from gemstones

### PR DESCRIPTION
**Pull request by ChrSteinert**
This adds the section about Quadlet to the guide version of the Podman page.

Would you like this section to be further expanded for the guide version?

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/guides/containers/podman_guide.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ tags:
 
 # Introduction
 
-!!! note
+!!! note "Note"
```
Comment (sspencerwire) [inline docs/guides/containers/podman_guide.md:13]: Hi @ChrSteinert, I'm commenting inline here. There's no need to add the "Note" in quotes as you have done. It is an option for translators to add that in their language if they want within the Crowdin interface, however most have chosen *not* to do this, as in general, the admonition titles, even in English, are pretty self-explanatory. I'd prefer that you not add these within the English source documents.

File: docs/guides/containers/podman_guide.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Here is a non-exhaustive list of the most commonly used subcommands:
 | `unpause`   | Unpauses the processes in one or more containers                 |
 | `volume`    | Manages volumes                                                  |
 
-!!! note
+!!! note "Note"
```
Comment (sspencerwire) [inline docs/guides/containers/podman_guide.md:87]: See mention above on this

**Comment by sspencerwire** [comment]
@ChrSteinert If there's more to be said within the guide that will be helpful, please feel free to add it. Thank you!

**Comment by ChrSteinert** [comment]
Let me have a go through it for a few days maybe! :)

**Comment by ChrSteinert** [comment]
That's what I have for now.

**Comment by sspencerwire** [comment]
> That's what I have for now.

If you are ready for a review and merge @ChrSteinert , just take it out of draft mode.

**Comment by ChrSteinert** [comment]
Stupid me – thanks for the heads up!

**Review by sspencerwire — APPROVED** [review]
Thanks @ChrSteinert .. light editing to follow

**Pull request merged**